README.rst

```diff
@@ -13,8 +13,8 @@ enterprise
     :target: http://codecov.io/github/edx/edx-enterprise?branch=master
     :alt: Codecov
 
-.. image:: http://edx-enterprise.readthedocs.io/en/latest/?badge=latest
-    :target: http://edx-enterprise.readthedocs.io/en/latest/
+.. image:: http://open-edx-enterprise-service-documentation.readthedocs.io/en/latest/?badge=latest
+    :target: http://open-edx-enterprise-service-documentation.readthedocs.io/en/latest/
```

Comment:
Previously broken link. Since I'm updating a bunch of stuff, thought I'd touch this.